modules/raft/src/main/java/org/apache/ignite/raft/jraft/entity/PeerId.java

```diff
@@ -287,6 +287,9 @@ else if (!this.endpoint.equals(other.endpoint)) {
     }
 
     public static PeerId fromPeer(Peer p) {
```

Comment:
It's look like a factory method, so it's confusing that it could produce null.
Are there any use-cases of null p Peer?
In any case lets add javadoc and @Nullable/@NotNull annotations.

Reply:
Use case here https://github.com/apache/ignite-3/pull/321/files#diff-bfb2b9f4058e3034104dbd6a860a201f04ac4066c2b2411bd1f717e23c26ab4aR128 - I thought it will be useful to support transparent null pointer processing here.